src/sentry/sentry_apps/apps.py

```diff
@@ -84,7 +84,7 @@ class SentryAppUpdater:
     overview: str | None = None
     allowed_origins: List[str] | None = None
     popularity: int | None = None
-    features: List[str] | None = None
+    features: List[int] | None = None
```

Comment:
Updated type to `List[int]` to match this one (the only usage of the `features` class field):
https://github.com/getsentry/sentry/blob/28c52c8e89e6ff6503559786b913c3e52623b7b0/src/sentry/models/integrations/integration_feature.py#L163